Performance.php

```diff
@@ -51,11 +51,7 @@
         $definitionResolver = new DefinitionResolver($index);
         $parser = new PhpParser\Parser();
 
-        try {
-            $document = new PhpDocument($testCaseFile, $fileContents, $index, $parser, $docBlockFactory, $definitionResolver);
-        } catch (\Throwable $e) {
-            continue;
-        }
+        $document = new PhpDocument($testCaseFile, $fileContents, $index, $parser, $docBlockFactory, $definitionResolver);
```

Comment:
Does the benchmark still run without this on all frameworks? I think I added this cause some files are invalid (on purpose, e.g. test fixtures) and this would break the whole loop

Reply:
It does indeed. The `(string)` thing fixes the test fixture thing in `math-php`. I also have (somehow by accident) downloaded Tolerant PHP Parser's validation frameworks, and it doesn't choke on them either.